demo/app/app_test.go

```diff
@@ -133,51 +132,99 @@ func TestInstantiateBabylonContracts(t *testing.T) {
 	btcFinalityContractCodeID := resp.CodeID
 	require.NoError(t, err)
 
-	initMsg, err := cli.ParseInstantiateArgs(
-		[]string{
-			fmt.Sprintf("%d", babylonContractCodeID),
-			fmt.Sprintf("%d", btcLightClientContractCodeID),
-			fmt.Sprintf("%d", btcStakingContractCodeID),
-			fmt.Sprintf("%d", btcFinalityContractCodeID),
-			"regtest",
-			"01020304",
-			"1",
-			"2",
-			"false",
-			fmt.Sprintf(`{"admin":"%s"}`, babylonKeeper.GetAuthority()),
-			fmt.Sprintf(`{"admin":"%s"}`, babylonKeeper.GetAuthority()),
-			"test-consumer",
-			"test-consumer-description",
-		},
-		"",
-		babylonKeeper.GetAuthority(),
-		babylonKeeper.GetAuthority(),
-	)
+	network := "regtest"
+	btcConfirmationDepth := 1
+	btcFinalizationTimeout := 2
+	babylonAdmin := consumerApp.BabylonKeeper.GetAuthority()
+	btcLightClientInitMsg := fmt.Sprintf(`{"network":"%s","btc_confirmation_depth":%d,"checkpoint_finalization_timeout":%d}`, network, btcConfirmationDepth, btcFinalizationTimeout)
+	btcFinalityInitMsg := fmt.Sprintf(`{"admin":"%s"}`, babylonAdmin)
+	btcStakingInitMsg := fmt.Sprintf(`{"admin":"%s"}`, babylonAdmin)
```

Comment:
How about creating some wrapper functions for creating instantiation msg json strings for contracts?

Reply:
These are only used in the demo and e2e tests, so the only common place would be the SDK. However, the SDK does not really know about the instantiation of the contracts anymore, so it would not be appropriate to place them there. Happy to hear alternatives